components/lvgl_esp32_drivers/lvgl_tft/rm67162Lilygo.c

```diff
@@ -217,7 +217,7 @@ void lcd_init(void)
                          lcd_init[i].len & 0x7f);
 
             if (lcd_init[i].len & 0x80)
-                vTaskDelay(120 / portTICK_RATE_MS);
+                vTaskDelay(120 / portTICK_PERIOD_MS);
```

Comment:
you should not change any part of the code out of your task scope

Reply:
because my project did not compile !